plugins/embed-optimizer/hooks.php

```diff
@@ -181,8 +181,9 @@ function embed_optimizer_update_markup( WP_HTML_Tag_Processor $html_processor, b
 		'script' => 'embed_optimizer_script',
 		'iframe' => 'embed_optimizer_iframe',
 	);
-	$trigger_error  = static function ( string $message ): void {
-		wp_trigger_error( __FUNCTION__, esc_html( $message ) );
+	$function_name  = __FUNCTION__;
+	$trigger_error  = static function ( string $message ) use ( &$function_name ): void {
```

Comment:
No need to pass by reference, right?

```suggestion
	$trigger_error  = static function ( string $message ) use ( $function_name ): void {
```

Reply:
sure, I'm not certain without testing but trust your intuition